docs/get_started/install.md

```diff
@@ -108,7 +108,7 @@ Installing *MXNet* with pip requires a latest version of `pip`. Install the late
 Install *MXNet* with OpenBLAS acceleration.
 
 ```bash
-(mxnet)$ pip install mxnet
+(mxnet)$ sudo pip install mxnet
```

Comment:
not sure about adding sudo by default. It could change installation path when you have a user python installed